app/src/main/java/kr/co/connect/boostcamp/livewhere/data/SharedPreferenceStorage.kt

```diff
@@ -0,0 +1,38 @@
+package kr.co.connect.boostcamp.livewhere.data
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.core.content.edit
+import kotlin.properties.ReadWriteProperty
+import kotlin.reflect.KProperty
+
+interface SharedPreference {
+    var uuid: String?
+}
+
+class SharedPreferenceStorage(context: Context) : SharedPreference {
+    private val prefs = context.applicationContext.getSharedPreferences(PREF, Context.MODE_PRIVATE)
+
+    override var uuid: String? by UuidPreference(prefs, PREF_UUID, null)
+
+    companion object {
+        const val PREF = "pref"
+        const val PREF_UUID = "pref_uuid"
+    }
+}
+
+class UuidPreference(
```

Comment:
`p3;` 제 경우 `by` 키워드를 사용하여 처리하는 경우 Suffix로 `Delegate`를 붙이는 것을 선호하는 편입니다.
```suggestion
class UuidPreferenceDelegate(
```